nri-config-generator/args/args.go

```diff
@@ -26,19 +25,15 @@ type ArgumentList struct {
 	ShortRunning bool   `default:"false" help:"By default execution is long running, but this can be override"`
 }
 
-func argVars(c *ArgumentList) (map[string]interface{}, error) {
-	err := args.SetupArgs(c)
```

Comment:
isn't it this converting from env variables to args? 
I think this helps to set configurations like verbose